src/Service.js

```diff
@@ -0,0 +1,39 @@
+const { Car } = require('./Car');
+
+class Service {
+  #cars;
+
+  constructor(names) {
+    this.#cars = Array.from(
+      { length: names.length },
+      (_, index) => new Car(names[index])
+    );
+  }
+
+  getMovingLog() {
+    const roundLog = {};
+
+    this.#cars.forEach((car) => {
+      car.move();
+      const { name, movingLog } = car.getCarInfo();
+      roundLog[name] = movingLog;
+    });
+
+    return roundLog;
+  }
+
+  getWinners() {
+    const cars = Array.from({ length: this.#cars.length }, (_, index) =>
+      this.#cars[index].getCarInfo()
+    );
+    const max = cars.sort((a, b) => b.movingLog - a.movingLog)[0].movingLog;
+    const winner = [];
+    cars.forEach((car) => {
+      if (car.movingLog === max) winner.push(car.name);
+    });
+
```

Comment:
다른 Array 메서드를 이용하여 구현 할 수 있을 것 같습니다~! 어떤 메서드를 사용할 수 있을까요?

Reply:
filter까지 추가해서 수정했습니다!
```suggestion
    const winner = cars
      .filter((car) => car.moveCnt === max)
      .map((car) => car.name);
    return winner;

```

Reply:
👍👍